### PR TITLE
Connecting profile and index page 

### PR DESCRIPTION
**Pull request by forfeen**

### CODE REVIEW

File: mysite/settings.py [hunk 1]
```diff
@@ -105,7 +107,7 @@
 
 LANGUAGE_CODE = 'en-us'
 
-TIME_ZONE = 'UTC'
+TIME_ZONE = config('TIME_ZONE', default='UTC')
 
 USE_I18N = True
```
Comment (bleachjade) [inline mysite/settings.py]: you should not expose your secret key to the public, try to config it and then keep it in .env file

File: Kvent/urls.py [hunk 2]
```diff
@@ -4,4 +4,5 @@
 
 urlpatterns = [
     path('', views.index, name='index'),
-]
\ No newline at end of file
+    path('profile', views.profile, name='profile')
```
Comment (bleachjade) [inline Kvent/urls.py]: keep this to redirected to profile page

File: Kvent/models.py [hunk 3]
```diff
@@ -1,3 +1,23 @@
...
+from django.contrib.auth.models import User
 
-# Create your models here.
+class Info(models.Model):
+    """User's model"""
+    user = models.OneToOneField(User, on_delete=models.CASCADE, unique=True, default=None)
+    phone_num = models.CharField('Phone Number', max_length=10, default="NOT SET")
+    email = models.EmailField('E-mail', max_length=254)
+    address = models.CharField("Address", max_length=125, default="NOT SET")
+
```
Comment (forfeen) [inline Kvent/models.py:12]: Delete unnecessary space

File: mysite/settings.py (hunk 1 above)
Comment (forfeen) [reply mysite/settings.py]: I already configured it.

**Pull request merged**